.github/workflows/mac.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         submodules: recursive
 
     - name: setup Homebrew
-      run: brew install hidapi libxslt libjpg libmtp create-dmg confuse automake
+      run: brew install hidapi libxslt libjpg libmtp libraw create-dmg confuse automake
```

Comment:
Does this actually result in the entire libraw library being included in the .dmg that is built, or does this just add the library stubs so that Subsurface is built against libraw?
i.e. what happens when a version of Subsurface built in this way is run on a Mac that does not have the homebrew `libraw` installed when the user tries to add a RAW image?
The same applies for other distribution channels like COPR as well, where I think we need to declare a dependency on libraw in the manifest.

Reply:
macOS binaries are very much like AppImages or Snaps. They contain all of the libraries that they depend on (with the exception of what is considered 'core system libraries'). So this treats `libraw` just like we treat `libxslt` or `libjpg` or `libpng`. They all get bundled with the `Subsurface.app`

Reply:
TIL